dask_searchcv/tests/test_adaptive.py

```diff
@@ -0,0 +1,179 @@
+from dask_searchcv.adaptive import _top_k, Hyperband
+from dask_ml.linear_model import PartialSGDClassifier
+from dask_ml.datasets import make_classification
+import numpy as np
+import dask.array as da
+from dask.distributed import Client
+import pandas as pd
+import pytest
+import distributed
+from pprint import pprint
+import scipy.stats as stats
+import random
+from sklearn.linear_model import Lasso
+
+
+class TestFunction:
+    def _fn(self):
+        return self.value
+
+    def get_params(self, deep=None, **kwargs):
+        return {k: getattr(self, k) for k, v in kwargs.items()}
+
+    def set_params(self, **kwargs):
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+        return self
+
+    def partial_fit(self, *args, **kwargs):
+        pass
+
+    def score(self, *args, **kwargs):
+        return self._fn()
+
+
+def test_hyperband_sklearn():
+    client = _get_client()
+    X, y = make_classification(n_samples=1000, chunks=500)
+    classes = da.unique(y)
```

Comment:
This will always be `[0, 1]`. It's controlled by the `n_classes` argument to `make_classification`, and currently only `2` is supported.